R/run_app.R

```diff
@@ -12,24 +21,44 @@ run_app <- function(
   options = list(),
   enableBookmarking = NULL,
   uiPattern = "/",
+  app_ver = NULL,
+  login_note = NULL,
+  credentials_db_name = NULL,
+  assessment_db_name = NULL,
   ...
 ) {
+  # Pre-process some run-app inputs
+  if(is.null(app_ver)) app_ver <- paste0(packageVersion("riskassessment"))
+  if(is.null(assessment_db_name)) assessment_db_name <- "database.sqlite"
+  if(is.null(credentials_db_name)) credentials_db_name <- "credentials.sqlite"
+  if(is.null(login_note) & !file.exists(credentials_db_name)) {
+    login_note <- HTML('<em>Note:</em> To log in for the first time, use the admin user:
```

Comment:
Yes, that actually was intentional. I was thinking that long term we'll probably only want to note the 'admin' account, since that's what you need to get started with the app... but I thought maybe we'd keep the 'nonadmin' account on there just for testing purposes? What do you think? Our future selves will probably be confused by this.

Reply:
Initial reaction is "a temporary solution becomes a long problem." It poses less risk than multiple admin accounts though. I'm good with merging as-is.

Reply:
eh, I'll just get rid of the nonadmin for now. Maybe when we tackle #278, we can create a 'default_admin' and 'default_nonadmin' account launched directly from `run_app()` for testing purposes (bypassing auth screen).

Reply:
I like that idea better. I think we have been under utilizing `{golem}` options and configurations. We should work to setup a development configuration.